hermes-consumers/src/main/java/pl/allegro/tech/hermes/consumers/config/Http2ClientProperties.java

```diff
@@ -0,0 +1,68 @@
+package pl.allegro.tech.hermes.consumers.config;
+
+import org.springframework.boot.context.properties.ConfigurationProperties;
+import pl.allegro.tech.hermes.consumers.consumer.sender.http.Http2ClientParameters;
+
+@ConfigurationProperties(prefix = "consumer.http2.client")
+public class Http2ClientProperties {
+
+    private boolean enabled = true;
```

Comment:
Do we want to change a path for this property? 
`consumer.http2.enabled` -> `consumer.http2.client.enabled`

Reply:
Yes, i would like to do it, just take a look at usage of this property: https://github.com/allegro/hermes/pull/1513/files#diff-4b2f6d36b9dd36cd1b7ba62b3bf9f3ad4b2e1a9dfce7b13d40a82f35d0926fbcR126